src/System/Envy.hs

```diff
@@ -312,7 +311,7 @@ setEnvironment' = setEnvironment . toEnv
 
 ------------------------------------------------------------------------------
 -- | Unset Environment from a `ToEnv` constrained type
-unsetEnvironment :: ToEnv a => EnvList a -> IO (Either String ())
+unsetEnvironment :: EnvList a -> IO (Either String ())
```

Comment:
GHC complained about it. I'm not familiar enough with Envy to say if it's logically necessary or not.